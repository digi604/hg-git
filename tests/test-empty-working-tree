#!/bin/sh

"$TESTDIR/hghave" git || exit 80

hg init
git init -q

git commit --allow-empty -qm empty

echo % original HEAD
git rev-list -n 1 HEAD

hg gimport
hg log -r tip --template 'files: {files}\n'

hg gclear
hg gexport

echo % exported HEAD
git rev-list -n 1 HEAD
