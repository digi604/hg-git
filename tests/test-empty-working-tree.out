% original HEAD
1f2153667658aafaa7e7700dddb6a0899c3f5a40
importing Git objects into Hg
at: 0/1
creating bookmarks failed, do you have bookmarks enabled?
files: 
clearing out the git cache data
importing Hg objects into Git
at: 0/1
converting revision 0
% exported HEAD
1f2153667658aafaa7e7700dddb6a0899c3f5a40
